pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2487,7 +2488,7 @@ The delayed message index bucket time step(in seconds) in per bucket snapshot se
     )
     private List<String> bootstrapNamespaces = new ArrayList<String>();
 
-    @ToString.Exclude
```

Comment:
I wonder why this lombok anonotion can not take affect

Reply:
The current implementation uses ReflectionToStringBuilder to print the serialized result. If I understand correctly, ReflectionToStringBuilder.toString(config) does not recognize lombok's annotations, we need to use the corresponding annotations to filter privacy information, namely StringToExclude instead of lombok's String.ToExclude

> ... Alternatively the ToStringExclude annotation can be used to exclude fields from being incorporated in the result. ... public class ReflectionToStringBuilder extends ToStringBuilder { ...

Reply:
Thanks for your explain. now I see it. The root cause is that we didn't over write to `toString()` method of `ServiceConfiguration`.

How about just change it like this:
```
@ToString
@Getter
@Setter
public class ServiceConfiguration
```
or
```
@Data
public class ServiceConfiguration
```

because it looks that we need the `toString()` method.

Reply:
Thanks for your advice. I tried to add ToString annotation and it works as you expected.
I'm not sure if @Data is ok as it is equivalent to @Getter @Setter @RequiredArgsConstructor @ToString @EqualsAndHashCode.
If I understand correctly. We just need@Getter @Setter and @ToString only.